app/components/calls_to_action/adviser_component.rb

```diff
@@ -0,0 +1,19 @@
+class CallsToAction::AdviserComponent < ViewComponent::Base
+  attr_reader :title, :text, :image, :link_text, :link_target
+
+  def initialize(
+    title: "Get free one-to-one support",
+    text: "An adviser with years of teaching experience can help you to become a maths teacher. Chat by phone, text, or email as little or often as you need.",
```

Comment:
```suggestion
    text: "An adviser with years of teaching experience can help you to become a teacher. Chat by phone, text, or email as little or often as you need.",
```
@martyn-w I presume this is the default text that can be overwritten in the front matter of a particular page as needed?

If so, I suggest we make the default text more generic in case it is used by accident!